app/sidekiq/EventReminderEmail.rb

```diff
@@ -0,0 +1,7 @@
+class EventReminderEmail
+  include Sidekiq::Job
+
+  def perform(users, event)
+    users.each { |user| UserMailer.mail_event_reminder(user, event).deliver }
```

Comment:
Very nice. Not related to this PR, but can you build in throttling? Something as simple as a sleep of ~10/15s should be sufficient to please the Gmail gods.

Reply:
I see that there is a different PR for this (#501). Up to you to do it in this PR or #501.